src/tool/pmix_tool.c

```diff
@@ -664,7 +664,14 @@ PMIX_EXPORT pmix_status_t PMIx_tool_finalize(void)
     if (0 <= pmix_client_globals.myserver.sd) {
         CLOSE_THE_SOCKET(pmix_client_globals.myserver.sd);
     }
-    event_base_free(pmix_globals.evbase);
+/*
```

Comment:
Agreed - you can just delete this code (I may have already done this in a recent commit)